packages/mobile/src/invite/saga.ts

```diff
@@ -116,10 +116,6 @@ export function* sendInvite(
     const temporaryWalletAccount = web3.eth.accounts.create()
     const temporaryAddress = temporaryWalletAccount.address
     const inviteCode = createInviteCode(temporaryWalletAccount.privateKey)
-
-    // TODO: Improve this by not checking specifically for this
-    // display name. Requires improvements in recipient handling
-    recipientName = recipientName === i18n.t('sendFlow7:mobileNumber') ? '' : ' ' + recipientName
```

Comment:
Why remove this? We need it to prevent sending SMS messages with the text "Hi Mobile #"

Reply:
Don't really know how this ended up here, will restore